readme.md

```diff
@@ -1,151 +1,131 @@
-- Open source (installable via npm/Yarn or Docker Hub) so can be self-hosted or used via the Mend Renovate App
+Renovate can provide updates for most popular languages, platforms, and registries including: npm, Java, Python, .NET, Scala, Ruby, Go, Docker and more.
+Supports over 90 different package managers.
 
-## Supported Platforms
+### Platforms
 
-Renovate works on these platforms:
+Renovate updates code repositories on the following platforms: GitHub, GitLab, Bitbucket, Azure DevOps, AWS Code Commit, Gitea, Forgejo, Gerrit (experimental)
 
-- [GitHub (.com and Enterprise Server)](https://docs.renovatebot.com/modules/platform/github/)
-- [GitLab (.com and CE/EE)](https://docs.renovatebot.com/modules/platform/gitlab/)
-- [Bitbucket Cloud](https://docs.renovatebot.com/modules/platform/bitbucket/)
-- [Bitbucket Server](https://docs.renovatebot.com/modules/platform/bitbucket-server/)
-- [Azure DevOps](https://docs.renovatebot.com/modules/platform/azure/)
-- [AWS CodeCommit](https://docs.renovatebot.com/modules/platform/codecommit/)
-- [Gitea and Forgejo](https://docs.renovatebot.com/modules/platform/gitea/)
-- [Gerrit (experimental)](https://docs.renovatebot.com/modules/platform/gerrit/)
+## Ways to Run Renovate
 
-## Who Uses Renovate?
+The most effective way to run Renovate is to use an automated job scheduling system that regularly runs Renovate on all enabled repositories and responds with priority to user activity.
+Mend offers cloud-hosted and self-hosted solutions. See the options below.
 
-Renovate is widely used in the developer community:
+### Mend Renovate Community (Cloud-Hosted)
 
-![Logos of companies and projects that use Renovate](./docs/usage/assets/images/matrix.png)
+**Supports: GitHub.com, Bitbucket Cloud**
 
-<details>
-<summary>List of companies and projects that use Renovate</summary>
+Hosted by Mend.io. No setup is required. Community plan available (Free)
 
-- Prisma
-- Netlify
-- Envoy
-- Condé Nast
-- Microsoft
-- Atlassian
-- Sourcegraph
-- Mozilla
-- Deloitte
-- Telus
-- Yarn
-- HashiCorp
-- Automattic
-- Algolia
-- eBay
-- Cypress
-- Red Hat
-- Financial Times
-- Uber
-- Buildkite
+- GitHub Cloud: Install the [Renovate Cloud-Hosted App](https://github.com/apps/renovate) on your GitHub org, then select the repos to enable
+- Bitbucket Cloud: Add the [Mend App](https://marketplace.atlassian.com/apps/1232072/mend) to your Workspace, the add the Mend Renovate user to the projects you want to enable
 
-</details>
+### Mend Renovate Community (Self-hosted)
 
-## Renovate OSS Insights
+**Supports: GitHub, GitLab, Bitbucket Data Center**
 
-Renovate is built on a big community and actively invites and supports contributions.
-Information about our contributors and community can be found on [OSS Insight](https://ossinsight.io/analyze/renovatebot/renovate).
+Install and run your own Renovate server. Access internal packages.
```

Comment:
```suggestion
Install and run your own Renovate server.
Access internal packages.
```

Reply:
I am going for a shorter version (less page height), so prefer not to add additional line breaks.